### PR TITLE
[Poromechanics] Fixing fracture propagation utility and updating to current input standard

### PR DESCRIPTION
**Pull request by ipouplana**
This fixes the outdated files of PR #2360. Other minor cleanup changes are included.
Tests have also been updated and run flawlessly.

### CODE REVIEW

File: applications/PoromechanicsApplication/custom_utilities/poro_element_utilities.hpp [hunk 1]
```diff
@@ -904,7 +904,19 @@ typedef std::size_t IndexType;
     /// Rows: nodes
     /// Columns: GP
 
-    static inline void CalculateExtrapolationMatrix(BoundedMatrix<double,4,4>& rExtrapolationMatrix)
+    static inline void Calculate2DExtrapolationMatrix(BoundedMatrix<double,3,3>& rExtrapolationMatrix)
+    {
+        //Triangle_2d_3
+        //GI_GAUSS_2
+
+        rExtrapolationMatrix(0,0) = 1.6666666666666666421; rExtrapolationMatrix(0,1) = -0.33333333333333334443; rExtrapolationMatrix(0,2) = -0.33333333333333334443;
```
Comment (miguelmaso) [inline applications/PoromechanicsApplication/custom_utilities/poro_element_utilities.hpp]: should these constants be 5/3 and 1/3?

**Review by miguelmaso — DISMISSED** [review]
great!

File: applications/PoromechanicsApplication/custom_utilities/poro_element_utilities.hpp (hunk 1 above)
Comment (ipouplana) [reply applications/PoromechanicsApplication/custom_utilities/poro_element_utilities.hpp]: Thanks! I'll fix it

**Review by miguelmaso — APPROVED** [review]
Nice!

**Pull request merged**